src/content/docs/en/developers/verifying-smart-contracts.mdx

```diff
@@ -5,31 +5,39 @@ title: "Verifying Smart Contracts"
 lang: "en"
 permalink: "developers/verifying-smart-contracts"
 whatsnext: { "Scroll Contracts": "/developers/scroll-contracts" }
-excerpt: "Easily verify your smart contracts with Scroll-supported developer tooling or the Blockscout Web API"
+excerpt: "Easily verify your smart contracts with Scroll-supported developer tooling"
 ---
 
 import Aside from "../../../../components/Aside.astro"
 import ClickToZoom from "../../../../components/ClickToZoom.astro"
 import verify1 from "./_images/verify1.png"
 import CodeSample from "../../../../components/CodeSample/CodeSample.astro"
 
-After deploying your smart contracts, it's important to verify your code on [our block explorer](https://scrollscan.com/) or the [Sepolia block explorer](https://sepolia-blockscout.scroll.io).
+After deploying your smart contracts, it's important to verify your code on our block explorers. This can be done in an automated way using your developer tooling or the Web UI.
```

Comment:
I'm wondering if "our" is right if we're not running Scrollscan and other blockexplorers might want visibility on this page.

"on a block explorer"